pymc/ode/__init__.py

```diff
@@ -13,5 +17,6 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+
```

Comment:
```suggestion
```
The pre-commit will probably complain about this added empty line

Reply:
Change incorporated